bmw_cd_exporter.py

```diff
@@ -94,6 +106,14 @@ def parse_args():
         help='Override path to the JSON file containing all attributes (optional)',
         default='attributes.json'
     )
+    parser.add_argument(
+        '--mqtt',
+        metavar='x.y.z.w',
+        required=False,
+        type=str,
+        help='Mqtt server address',
+        default=os.environ.get('BMW_MQTTSERVER')
```

Comment:
Probably need an underscore between `MQTT` and `SERVER`

Reply:
Correct